src/styles/blocks/card.scss

```diff
@@ -0,0 +1,69 @@
+.card {
+  font-family: $main-font-family;
+  @include text(#616070, 10px, 400);
+  width: $card-width;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    padding: 0 3px;
+    width: 100%;
+    height: 134px;
+    margin-bottom: 38px;
+    object-fit: contain;
+  }
+
+  &__title {
+    @include text($color-dark-blue, 12px, 500);
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+    line-height: 14px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    text-align: right;
+    color: $color-dark-blue;
+  }
+
+  &__price-section {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__price {
+    @include text($color-dark-blue, 16px, 700);
+  }
+
+  &__buy-link {
+    display: block;
+    text-transform: uppercase;
+    text-decoration: none;
+    @include text(white, 14px, 700);
+    text-align: center;
+    padding: 11px 0;
+    background: $color-cyan;
+    border: 1px solid $color-cyan;
+    border-radius: 5px;
+  }
+
+  &__buy-link:hover {
+    color: $color-cyan;
+    background: white;
```

Comment:
set background-color instead of background

Reply:
Done.